spinta/cli/push.py

```diff
@@ -589,23 +929,329 @@ def _save_push_state(
     conn = context.get('push.state.conn')
     for row in rows:
         table = metadata.tables[row.data['_type']]
-        if row.saved:
+
+        if row.error and row.op != "delete":
+            data = json.dumps(row.data)
```

Comment:
I tink, we need to call `fix_data_for_json` as in other places, to convert things like dates to json serializable form.